crates/services/relayer/src/mock_db.rs

```diff
@@ -25,6 +29,8 @@ use std::{
 #[derive(Default)]
 pub struct Data {
     pub messages: BTreeMap<DaBlockHeight, HashMap<Nonce, Message>>,
+    pub transactions:
+        BTreeMap<DaBlockHeight, HashMap<RelayedTransactionId, RelayedTransaction>>,
```

Comment:
Why is `BTreeMap` used here? Do we need this to be sorted by `DaBlockHeight`?

Reply:
This is just a mockdb, so I don't think it really matter a whole lot. Was just following the convention we already had for the mock messages table.